backend/baseclass.pm

```diff
@@ -991,6 +991,7 @@ sub new_ssh_connection {
         }
         else {
             bmwqemu::diag "Could not connect to $args{username}\@$args{hostname}, Retry";
+            bmwqemu::diag "Possible solutions: check network-settings e.g. VIRSH_CMDLINE or S390_NETWORK_PARAMS and VNC/SSH settings in there";
```

Comment:
I will move this to line 1000, as the connection might not be ready when the loop starts, and only show it when it dies.